main.cpp

```diff
@@ -230,14 +236,14 @@ std::string CoffeeShop::takeOrder()
     return "May I take your order?";
 }
 
-std::string CoffeeShop::makeCoffee(std::string order1)
+void CoffeeShop::makeCoffee()
 {
-    return "I have made 1 " + order1;
+    std::cout << "I have made 1 " + CoffeeShop::order << "\n";
```

Comment:
you do not need to fully qualify member variables inside member functions.
- [ ] revise